validation/src/test/scala/hmda/validation/rules/lar/validity/V355Spec.scala

```diff
@@ -0,0 +1,58 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.{ Failure, Success }
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V355Spec extends LarEditCheckSpec with BadValueUtils {
+
+  property("If agency code is valid, lar must pass") {
```

Comment:
This property wording is confusing, since it seems to be implying different logic from the edit check. Could it change to "lar with valid denial reasons must pass"?

Same with the next one on line 19.